apphub/NLP/neural_machine_translation/transformer.ipynb

```diff
@@ -654,13 +899,13 @@
      "output_type": "stream",
      "text": [
       "Source Language: \n",
-      " muito obrigada. \n",
+      " portanto, o que esse servico faz, de forma simples, e deixar - vos criar uma mensagem ou video que pode ser colocada no facebook depois de morrerem. \n",
       "\n",
       "Translation Ground Truth: \n",
-      "thank you so much. \n",
+      "so what that service does, quite simply, is let you create a message or a video that can be posted to facebook after you die. \n",
       "\n",
       "Machine Translation: \n",
-      "thank you very much. \n"
+      "so what that service does, in simple, is to you stop a message or video video that can be put in you after you.. \n"
```

Comment:
is there a way we can keep the old example text instead?

Reply:
Yes, keep the old text :)